reference/orders.v2.oas2.yml

```diff
@@ -2961,6 +2995,8 @@ components:
           type: string
           nullable: true
           readOnly: true
+        fulfillment_source:
```

Comment:
If someone could provide the description for this field that would be great

Reply:
this field is added in "https://bigcommercecloud.atlassian.net/browse/ORDERS-1586", the description in the PR is "where the order product is being fulfilled, as part of FBA initiative".

@bc-vincent-zhao are we deprecating this field ? seems like you removed a lot of usage for the FBA logic ?

Reply:
^Also, for the description, do you mean the location name (for where the order product is being fulfilled), or the location code?

Reply:
as @donald-nguyen-bc suggested, I don't think this field is being used by merchants at all, so if it's not on the doc, let's not add it now.

we will create a ticket to remove it entirely from the api output.

Reply:
Thanks @bc-vincent-zhao  & @donald-nguyen-bc , I removed the field everywhere from the yml file